records_mover/db/redshift/unloader.py

```diff
@@ -84,10 +84,9 @@ def unload_to_s3_directory(self,
             #
             register_secret(aws_creds.token)
             register_secret(aws_creds.secret_key)
-            select = text(
-                "SELECT * FROM "
-                f"{quote_schema_and_table(None, schema, table, db_engine=self.db_engine)}")
-            unload = UnloadFromSelect(select=select,
+            table_obj = Table(table, MetaData(), schema=schema)
+            select = str(sqlalchemy.select('*', table_obj))  # type: ignore[arg-type]  # noqa: F821
```

Comment:
i don't think you have to str() this if you are calling text() on it below right?

Reply:
As above, need to convert to string to pass to text